### DISCUSSION TITLE
Submodules via YAML Key?

### DISCUSSION THREAD

**Discussion by ArneNostitz** [Vinzent03/obsidian-git#427] [General]
hi. i am using obsidian as my zentral Tool for everything. i also manage my projects in there. and all relating contacts, notes and tasks. 

this was great until I added team members to one project. now i have to somehow share notes relating to this one project. files are not managed in folders though. because there a files related to more than one project. best example is a contact related to more than one project. 

any way you are solving a problem like this? 

is there a way to use a yaml key for example (like the publish tools are using) to sync those marked files to a second repository?

thanks. i am trying to solve this for nearly a month now but no solution is really good. 

but I guess i am not the only one having a problem like this? :-)

happy for any tip!

thanks

**Comment by Vinzent03** [comment]
I don't plan on adding this, because this requires a lot of working around natural Git behavior. You may initialize two Git repositories in the same directory and declare one as your default in the settings. You can then use the other repo via command line. Not only that, but you would have to manage the file ignoring somehow.
See [here](https://stackoverflow.com/questions/436125/two-git-repositories-in-one-directory) for a bit of explanation.

**Reply by ArneNostitz** [reply]
thanks for the tip - yes i realize its somehow complicated as Git does have an ignore but not a e"include" file ... that would be great as that could add all files marked with a key ... but I do fear that i will have to separate out the files project related somehow ...
